src/tfp_backend/Transform_mir.ml

```diff
@@ -10,11 +10,21 @@ let remove_stan_dist_suffix s =
     ~f:(fun suffix -> String.chop_suffix ~suffix s)
   |> List.hd_exn
 
-let capitalize_fnames = String.Set.of_list ["normal"; "cauchy"]
+let capitalize_fnames =
+  String.Set.of_list ["normal"; "cauchy"; "gumbel"; "exponential"; "gamma"]
+
+(* Bernoulli, Binomial, NegativeBinomial should be mapped to TFP with either probs= or logits= flags *)
+(* Poisson should be mapped to TFP with either rate or log_rate *)
+(* | "lkj_corr_cholesky" -> ("CholeskyLKJ", args) *)
```

Comment:
Why is this one commented out?

Reply:
`| "lkj_corr_cholesky" -> ("CholeskyLKJ", args)` is commented out because TFP's `CholeskyLKJ` takes two parameters (dim and concentration) whereas args only contains concentration. Perhaps there's some way to pull the dim from `cholesky_factor_corr[K] L` into `args`?

Regarding `Bernoulli`, `Binomial` etc - as far as I understand, we want to map `binomial_logit(N, alpha)` to `Binomial(N, logits=alpha)` and `binomial(N, theta)` to `Binomial(N, probs=theta)`. Here, as well, I wasn't sure how to do this so I commented it out for now...

Reply:
One day we might like to have a different tree structure than MIR for printing Python, one that  supports keyword arguments and maybe inline function definitions. For now I think it might be okay to essentially use those as positional arguments by inserting `None` (or whatever the default is as appropriate.

Reply:
One possible workaround is to define "helper" function in the generated .py file, something like
```python
binomial_logit = lambda n,a : tfd__.Binomial(n, logits=a)
binomial = lambda n,a : tfd__.Binomial(n, probs=a)
```
and then add
```ocaml
| "binomial_logit" -> ("binomial_logit", args)
| "binomial" -> ("binomial", args)
```
This will make the .py file uglier, but should allow us to extend the number of supported distributions without making significant changes to the compiler itself; and removing it once such changes will happen should be straightforward. What do think?